.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,48 @@
+# Chromatic Effects on PSF and Shear Measurement for Roman
+
+## Introduction
+
+The software and data used are based on the paper:
+ - Chromatic Effects on the PSF and Shear Measurement for the Roman Space Telescope High-Latitude Wide Area Survey; Berlfein et al. (2025), submitted - [ADS entry](https://ui.adsabs.harvard.edu/abs/2025arXiv250500093B/abstract)
+
+
+This repository includes software to:
+ - Generate Roman-like image simulations of the [OpenUniverse2024 catalog](https://irsa.ipac.caltech.edu/data/theory/openuniverse2024/overview.html) or the [cosmoDC2 catalog](https://portal.nersc.gov/project/lsst/cosmoDC2/_README.html) using [GalSim](https://github.com/GalSim-developers/GalSim).
+ - Generate catalog-level magnitude noise using [PhotErr](https://github.com/jfcrenshaw/photerr).
+ - Perform shear measurement on galaxy images using [AnaCal](https://github.com/mr-superonion/AnaCal).
+ - Correct for SED-dependant chromatic effects on the PSF.
+
+Note that in order to generate the actual image simulations, the relevant extragalactic and stellar catalogs are not provided here and must be downloaded. We provide catalog-level magnitudes (both true and observed), along with their errors, for the Roman (YJHFW) and LSST (ugrizy) bands. All entries contain object IDs in case you wish to cross-match with the full simulated catalogs. 
+
+
+## Installation
```

Comment:
Installation is not the same as cloning.  When we talk about installation, usually that's something that is done with a library, e.g., using pip, that results in an ability to import the library.  I would suggest changing this heading to something like "Using this software" and have an explanatory sentence that there is nothing to install, but rather a set of notebooks and scripts that people can run after cloning the repo.